### ISSUE TITLE
Video on android not preview

### ISSUE THREAD

**Issue by febobo** [TheWidlarzGroup/react-native-video#262]
Video on android not preview

**Comment by kayrules** [comment]
yes, I also had this problem. No issue on IOS. Any tips to fix this problem on android?
RN 0.25.

**Comment by febobo** [comment]
I now not solve program.. if you solve. please reply me

**Comment by superandrew213** [comment]
@kayrules @febobo for android use `<Image source={{ uri: videoUri }} />`

**Comment by superandrew213** [comment]
Another way to get the preview to show is by doing this

```
            <Video
              ref={ref => this._video = ref}
              source={{ uri: videoUri }}
              style={styles.video}
              resizeMode={'cover'}
              repeat={true}
              onLoad={() => {
                this._video.seek(0);
              }}
            />
```

**Comment by febobo** [comment]
@superandrew213  thanks.  Image Component I already try.  as if don't work. I well try ref

**Comment by superandrew213** [comment]
@febobo image component should work. I just used it in my project. Make sure you set width and height of image component

```
<Image source={{ uri: videoUri }} style={{ height: 100, width: 100 }} />
```

**Comment by febobo** [comment]
@superandrew213  I now, according to the method that you provided. do not work. I react native version is 0.28
![image](https://cloud.githubusercontent.com/assets/9276376/17015264/c091e4b8-4f5b-11e6-8749-7a5f04bf0a4c.png)
![image](https://cloud.githubusercontent.com/assets/9276376/17015269/cb18749c-4f5b-11e6-9367-7bfa18e0789c.png)

**Comment by superandrew213** [comment]
@febobo not sure why it is not working for you. I am using RN 0.26.2 but doubt that, that would be the issue. Does the video play? Did you also try `this._video.seek(1);`?

**Comment by antonydieu** [comment]
I user : react-native: 0.30.0
In file **index.android.js**
import { 
    Video
    } from 'react-native-video';
 **And**
let pic = {
      uri: 'https://ds-vn.serving-sys.com/BurstingRes/Site-16969/Type-16/fa63cea8-4249-42f9-b45d-5b431270c49c.mp4'
    };
 return (

```
  <Video source={pic} // Can be a URL or a local file.
   rate={1.0}                   // 0 is paused, 1 is normal.
   volume={1.0}                 // 0 is muted, 1 is normal.
   muted={false}                // Mutes the audio entirely.
   paused={false}               // Pauses playback entirely.
   resizeMode="cover"           // Fill the whole screen at aspect ratio.
   repeat={true}                // Repeat forever.
   playInBackground={false}     // Audio continues to play when app entering background.
   playWhenInactive={false}     // [iOS] Video continues to play when control or notification center are shown.
   onLoadStart={this.loadStart} // Callback when video starts to load
   onLoad={this.setDuration}    // Callback when video loads
   onProgress={this.setTime}    // Callback every ~250ms with currentTime
   onEnd={this.onEnd}           // Callback when playback finishes
   onError={this.videoError}    // Callback when video cannot be loaded
   style={styles.backgroundVideo} />
);
```

**Add it to your project**

**Run:**  npm install react-native-video --save
rnpm link react-native-video

**android/settings.gradle**

include ':react-native-video'
project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android')

**android/app/build.gradle**

dependencies {
   ...
   compile project(':react-native-video')
}

**And in MainApplication**
 @Override
        protected List<ReactPackage> getPackages() {
            return Arrays.<ReactPackage>asList(
                    new MainReactPackage(), new ReactVideoPackage()
            );
        }

But I get 
![screenshot_2016-08-05-17-03-55](https://cloud.githubusercontent.com/assets/5988428/17433460/eb69baf6-5b2e-11e6-831e-d7072258b672.png)

Please help me!

**Comment by hamazy** [comment]
Just FYI. I also had the same problem with RN 0.31.0 and `seek(0)` solved it perfectly.

**Comment by mattapperson** [comment]
Closing as this seems to be fixed now in RN 0.31. Please feel free to re-open if your still having issues

**Comment by America-first-melon** [comment]
mark
